src/globals.d.ts

```diff
@@ -1,3 +1,192 @@
 declare function escape(s: string): string;
 declare function unescape(s: string): string;
 declare module 'duo_web_sdk';
+
+declare module "@nodert-win10-rs4/windows.security.credentials" {
```

Comment:
Did you generate these typing somehow?

Reply:
The typings within the package uses the name `windows.security.credentials` which were not picked up. I did some quick searching but seems like there is no way to create an "alias". Not sure why the package provides typing without the prefix.